tests/test_challenges.py

```diff
@@ -0,0 +1,64 @@
+import ast
+import click
+import responses
+
+from click.testing import CliRunner
+from pylsy import pylsytable
+
+from evalai.challenges import challenges
+from tests.data import challenge_response
+
+from evalai.utils.challenges import API_HOST_URL
+from evalai.utils.urls import urls
+
+
+class TestChallenges:
+
+    def setup(self):
+
+        json_data = ast.literal_eval(challenge_response.challenges)
+
+        url = "{}{}"
+        responses.add(responses.GET, url.format(API_HOST_URL, urls["get_challenge_list"]),
```

Comment:
Please update URLs here as well.
TDD for the win, always. :slightly_smiling_face: